src/nodes/chunk_append/chunk_append.c

```diff
@@ -195,6 +195,80 @@ ts_chunk_append_path_create(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht,
 			break;
 	}
 
+	if (!ordered || ht->space->num_dimensions == 1)
```

Comment:
shouldn't this have different shape at toplevel:
```
if (!ordered)
{
  // no ordered append
} else if (ht->space->num_dimensions == 1) {
  // ordered append with only 1 dimension
} else {
  // ordered append with multiple dimensions
}
```

Reply:
fixed